src/js/tracks/text-track-list.js

```diff
@@ -89,15 +89,22 @@ TextTrackList.prototype.addTrack_ = function(track) {
 };
 
 TextTrackList.prototype.removeTrack_ = function(rtrack) {
-  let result = null;
   let track;
 
   for (let i = 0, l = this.length; i < l; i++) {
-    track = this[i];
-    if (track === rtrack) {
-      this.tracks_.splice(i, 1);
-      break;
+    if (this[i] !== rtrack) {
+      continue;
```

Comment:
Can we keep it so that it only does the splice and break if they match instead of continuing? It'll be more inline with how everything else is written.